tests/php/test_fieldmanager_field.php

```diff
@@ -1078,4 +1078,46 @@ public function test_element_markup_filters() {
 		$fm->element_markup( rand_str() );
 		$this->assertSame( 4, $ma->get_call_count(), 'Missing calls to element-markup filters' );
 	}
+
+	public function test_display_if_markup_for_page_template() {
+		$name     = rand_str();
+		$template = 'toaster.php';
```

Comment:
I'm sure there's a good story behind this.

Reply:
😄 

I wish it was more than this being a word unlikely to come up in a non-testing context. Instead of `foo` and `bar`, I use `waffle iron` and `toaster pastry`, for reasons I can no longer recall; probably something about being self-taught and learning programming when I was a hungry teenager. 😂